variables/_media-queries.pcss

```diff
@@ -1,3 +1,17 @@
+@custom-media --tec-viewport-xxsmall (min-width: 320px);
+@custom-media --tec-viewport-xsmall (min-width: 500px);
+@custom-media --tec-viewport-small (min-width: 600px);
+@custom-media --tec-viewport-medium (min-width: 768px);
+@custom-media --tec-viewport-wpadmin (min-width: 783px);
+@custom-media --tec-viewport-full (min-width: 960px);
+@custom-media --tec-viewport-large (min-width: 1200px);
+@custom-media --tec-viewport-xlarge (min-width: 1260px);
+
+/* -----------------------------------------------------------------------------
+ * For backwards Compatibility with plugins that have not been updated yet.
+ * Remove these once all plugins have been updated to use the namespaced vars.
+ * ----------------------------------------------------------------------------- */
+
```

Comment:
we don't need to namespace these, we'll add a flag into the compiler to resolve these rather than preserve them.